expressEngine/lib/service/db.js

```diff
@@ -27,6 +27,22 @@ class DBOperation {
     }
   }
 
+  updateOne = async (collection, dataToUpdate, filter) => {
+    try {
+      return await collection.updateOne(filter, { $set: dataToUpdate });
```

Comment:
@aman-successive please give the reason why you are using $set when it is even working without it.

Reply:
As you must be knowing, $set is used in the update method to ensure that if the specified field is not there in the database, it will add that field along with the value, If the field is present, it will just update the value